src/parsers/parser.js

```diff
@@ -45,6 +45,11 @@ class Parser {
         return token && token.type === constants.KEYWORD && (token.value === kw);
     }
 
+    /**
```

Comment:
This comment is reflecting the intent of the previous implementation and it is also not necessary as the method's name describes its function.